sky/backends/backend_utils.py

```diff
@@ -2079,6 +2079,7 @@ def _refresh_cluster(cluster_name):
                 force_refresh=True,
                 acquire_per_cluster_status_lock=True)
         except (exceptions.ClusterStatusFetchingError,
+                exceptions.CloudUserIdentityError,
```

Comment:
Good to catch this here! Just checking  - is this related to azure version bump?

Reply:
No. It is not directly related, but it is to avoid the problem where the whole `sky status -r` fails due to one of the azure cluster failed to get the user identity (not needed).